demos/python_demos/face_recognition_demo/README.md

```diff
@@ -0,0 +1,187 @@
+                        camera, default)
+  -o PATH, --output PATH
+                        (optional) Path to save the output video to
+  -no_show              (optional) Do not display output
+  -tl, --timelapse      (optional) Auto-pause after each frame
+  -cw CROP_WIDTH, --crop_width CROP_WIDTH
+                        (optional) Crop the input stream to this width
+  -ch CROP_HEIGHT, --crop_height CROP_HEIGHT
+                        (optional) Crop the input stream to this height
+
+Faces database:
+  -fg PATH              Path to the face images directory
+  --run_detector        (optional) Use Face Detection model to find faces on
+                        the face images, otherwise use full images.
+
+Models:
+  -m_fd PATH            Path to the Face Detection Adas or Retail model XML
+                        file
+  -m_lm PATH            Path to the Facial Landmarks Regression Retail model
+                        XML file
+  -m_reid PATH          Path to the Face Reidentification Retail model XML
+                        file
+  -m_hp PATH            Path to the Head Pose Estimation Retail model XML file
+
+Inference options:
+  -d_fd {CPU,GPU,FPGA,MYRIAD,HETERO}
+                        (optional) Target device for the Face Detection Retail
+                        model (default: CPU)
+  -d_lm {CPU,GPU,FPGA,MYRIAD,HETERO}
+                        (optional) Target device for the Facial Landmarks
+                        Regression Retail model (default: CPU)
+  -d_reid {CPU,GPU,FPGA,MYRIAD,HETERO}
+                        (optional) Target device for the Face Reidentification
+                        Retail model (default: CPU)
+  -d_hp {CPU,GPU,FPGA,MYRIAD,HETERO}
+                        (optional) Target device for the Head Pose Estimation
+                        Retail model (default: CPU)
+  -l PATH, --cpu_lib PATH
+                        (optional) For MKLDNN (CPU)-targeted custom layers, if
+                        any. Path to a shared library with custom layers
+                        implementations
+  -c PATH, --gpu_lib PATH
+                        (optional) For clDNN (GPU)-targeted custom layers, if
+                        any. Path to the XML file with descriptions of the
+                        kernels
+  -v, --verbose         (optional) Be more verbose
+  -pc, --perf_stats     (optional) Output detailed per-layer performance stats
+  -t_fd [0..1]          (optional) Probability threshold for face
+                        detections(default: 0.6)
+  -t_id [0..1]          (optional) Cosine distance threshold between two
+                        vectors for face identification (default: 0.3)
+  -exp_r_fd NUMBER      (optional) Scaling ratio for bboxes passed to face
+                        recognition (default: 1.15)
+```
+
+Example of a valid command line to run the application:
+
+Linux (`sh`, `bash`, ...) (assuming OpenVINO installed in `/opt/intel/computer_vision_sdk`):
```

Comment:
```suggestion
Linux (`sh`, `bash`, ...) (assuming OpenVINO installed in `/opt/intel/openvino`):
```

My understanding is that the default path has changed since R5

Reply:
No, it has not - either on Windows or Linux. There are still `intel/computer_vision_sdk_<version>` and `intel/computer_vision_sdk` link.

Reply:
I'm quite positive that it installed to `openvino` on my machine. Though, now that I'm thinking, it might be a release candidate. Ok, nvm, then (will need to update after the next release =) )